ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/edge_ports/leaf-server-port-channels.j2

```diff
@@ -60,6 +61,12 @@
 {%                     if adapter_qos_profile is arista.avd.defined %}
     service_profile: {{ adapter_qos_profile }}
 {%                     endif %}
+{%                     if adapter_ptp is arista.avd.defined %}
+    ptp:
```

Comment:
I don't think PTP is on port-channel level is it? I would think it was only for member ports.

Reply:
You are correct, not yet there with MLAG.